spec/lib/unified_health_data/service_spec.rb

```diff
@@ -63,11 +65,22 @@
         it 'returns only enabled test codes' do
           allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_ch_enabled, user).and_return(true)
           allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_sp_enabled, user).and_return(false)
+          allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_mb_enabled, user).and_return(false)
           labs = service.get_labs(start_date: '2024-01-01', end_date: '2025-05-31')
           expect(labs.size).to eq(1)
           expect(labs.first.attributes.test_code).to eq('CH')
         end
       end
+
+      context 'when MB Flipper is enabled' do
+        it 'would return MB test codes if present in the data' do
+          allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_ch_enabled, user).and_return(false)
+          allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_sp_enabled, user).and_return(false)
+          allow(Flipper).to receive(:enabled?).with(:mhv_accelerated_delivery_uhd_mb_enabled, user).and_return(true)
+          labs = service.get_labs(start_date: '2024-01-01', end_date: '2025-05-31')
+          expect(labs).to be_empty # No MB records in the test data
```

Comment:
Update labs_response.json to include an MB result so this can test for the existence of that result instead of empty.